ftva_lab_data/admin.py

```diff
@@ -2,10 +2,13 @@
 from simple_history.admin import SimpleHistoryAdmin
 from .models import SheetImport, ItemStatus
 
-admin.site.register(SheetImport, SimpleHistoryAdmin)
-admin.site.register(ItemStatus)
-
 
+@admin.register(ItemStatus)
 class ItemStatusAdmin(admin.ModelAdmin):
-    list_display = ("status", "display_name")
-    search_fields = ("status", "display_name")
+    list_display = ("status",)
+    search_fields = ("status",)
+
+
+@admin.register(SheetImport)
+class SheetImportAdmin(SimpleHistoryAdmin):
+    search_fields = ("id",)
```

Comment:
I think this needs to be `id__exact`.  Currently, it's doing an `icontains` search, so when I search for shorter id values like `3` I get thousands of records.  We need to be able to go to the specific id requested.
https://docs.djangoproject.com/en/5.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.search_fields